agent/mitm/handlers/HttpRequestHandler.ts

```diff
@@ -273,6 +279,7 @@ export default class HttpRequestHandler extends BaseHttpHandler {
     }
 
     if (serverToProxyResponse instanceof http.IncomingMessage) {
+      resourceSize += serverToProxyResponse.rawTrailers.map(s => s.length).reduce((a, b) => a + b, 0);
```

Comment:
This should not be included in the response body. I think we're really adding up "response body byte length". I am not in favor of adding a calculation that is not a true "bytes on the wire" size to this. In the case of trailers and headers, those compact into frames

Reply:
Hmmm I tend to agree with you, but will the `afterHttpResponse` callback be able to take into account _all_ headers?

Reply:
You'l have all the trailers in that object that gets passed in.. so you could try this kind of approach.

Reply:
Trailers are very rare, fwiw